apps/member-profile/app/routes/_profile.profile.integrations.tsx

```diff
@@ -0,0 +1,133 @@
+import { json, type LoaderFunctionArgs } from '@remix-run/node';
+import { useLoaderData } from '@remix-run/react';
+import { ExternalLink } from 'react-feather';
+
+import { Pill, Text } from '@oyster/ui';
+import { iife } from '@oyster/utils';
+
+import {
+  ProfileHeader,
+  ProfileSection,
+  ProfileTitle,
+} from '../shared/components/profile';
+import { ENV } from '../shared/constants.server';
+import { getMember } from '../shared/queries';
+import { ensureUserAuthenticated, user } from '../shared/session.server';
+
+export async function loader({ request }: LoaderFunctionArgs) {
+  const session = await ensureUserAuthenticated(request);
+
+  const id = user(session);
+
+  const member = await getMember(id)
+    .select(['githubId'])
+    .executeTakeFirstOrThrow();
+
+  const githubOauthUri = iife(() => {
+    const url = new URL('https://github.com/login/oauth/authorize');
+
+    url.searchParams.set('client_id', ENV.GITHUB_OAUTH_CLIENT_ID || '');
+
+    return url.toString();
+  });
+
+  return json({
+    isGithubConnected: !!member.githubId,
+    githubOauthUri,
```

Comment:
One small change I made here is just formatting the entire URL in the `loader` so that there's no work needed on the client.